### PR TITLE
📖 Copy bib files for tex export

### PR DESCRIPTION
**Pull request by fwkoch**

### CODE REVIEW

File: apps/cli/src/store/local/actions.ts [hunk 1]
```diff
@@ -611,3 +616,8 @@ export async function processSite(session: ISession, opts?: ProcessOptions): Pro
   }
   return true;
 }
+
+export function bibtexSomething(session: ISession, path: string) {
+  const cache = castSession(session);
+  console.log(cache.$externalReferences);
+}
```
Comment (rowanc1) [inline apps/cli/src/store/local/actions.ts]: Not sure if this is left over?

Comment (fwkoch) [reply apps/cli/src/store/local/actions.ts]: Haha ok. Hmmf

File: packages/jtex/src/jtex.ts [hunk 2]
```diff
@@ -145,6 +150,7 @@ class JTex {
       errorLogFn: errorLogger(this.session),
       warningLogFn: warningLogger(this.session),
     };
+    if (bibliography) frontmatter.bibliography = bibliography;
```
Comment (rowanc1) [inline packages/jtex/src/jtex.ts]: should we also clone this object? Probably not a big deal for our stuff, but probably best if we are making changes to a different doc.

File: packages/jtex/src/jtex.ts [hunk 3]
```diff
@@ -145,6 +150,7 @@ class JTex {
...
     const validatedDoc = validateTemplateDoc(frontmatter, templateYml?.doc || [], options, opts);
```
Comment (rowanc1) [inline packages/jtex/src/jtex.ts]: Can you just check if we should be destructuring frontmatter before changing the bib?
ie.
```
const validatedDoc = validateTemplateDoc({...frontmatter, bibliography: bibliography ?? frontmatter.bibliography }, templateYml?.doc || [], options, opts);
 ```

File: packages/jtex/src/jtex.ts (hunk 2 above)
Comment (fwkoch) [reply packages/jtex/src/jtex.ts]: I think the validation functions clone objects, but I did destructure before changing bib.

Comment (rowanc1) [reply packages/jtex/src/jtex.ts]: Nice!

**Pull request merged**